employees/views.py

```diff
@@ -439,28 +450,17 @@ def post(  # pylint: disable=unused-argument
     check_permissions(allowed_user_types=[CustomUser.UserType.MANAGER.name, CustomUser.UserType.ADMIN.name]),
     name="dispatch",
 )
-class ProjectReportDetail(UserIsManagerOfCurrentReportProjectMixin, UpdateView):
-    template_name = "employees/project_report_detail.html"
-    form_class = ReportForm
-    model = Report
-
-    def get_context_data(self, **kwargs: Any) -> dict:
-        context_data = super().get_context_data(**kwargs)
-        context_data["UI_text"] = ProjectReportDetailStrings
-        return context_data
+class ProjectReportDetail(UserIsManagerOfCurrentReportProjectMixin, ReportDetailBase):
+    template_post_url = "project-report-detail"
+    template_discard_url = "project-report-list"
+    url_pk = "project"
 
     def get_success_url(self) -> str:
```

Comment:
This can be also moved to parent class, because only difference is in "project-report-list" vs "author-report-list".

Reply:
good catch, didn't think of it ;)